lib/logstash/outputs/udp.rb

```diff
@@ -27,7 +28,14 @@ def register
     @socket = UDPSocket.new
 
     @codec.on_event do |event, payload|
-      socket_send(payload)
+      port = event.sprintf(@port)
+      begin
+        port = Integer(port)
+      rescue => e # ArgumentError (invalid value for Integer(): "foo")
+        logger.error("Failed to resolve port, dropping event", port: @port, event: event.to_hash, message: e.message)
```

Comment:
Did you evaluate to put the failing event into DLQ?

Reply:
Not really, but did have a chat about the feature and dropping was considered acceptable.